packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -75,12 +89,21 @@ export class Snackbar implements ComponentInterface {
     this.snackbarElement.removeEventListener('MDCSnackbar:closing', (e) =>
       this.handleSnackbarHide(e)
     );
+    this.snackbarElement.removeEventListener('mouseenter', () => {
+      if (this.timeout) clearTimeout(this.timeout);
+    });
+    this.snackbarElement.removeEventListener('mouseleave', () => {
+      this.configureTimeout();
+    });
   }
 
   private configureTimeout() {
```

Comment:
Maybe we should call it something like `setupTimeout` to make its functionality more clear? What do you think?

Reply:
Seems reasonable, I changed it.